crates/figma_import/src/toolkit_style.rs

```diff
@@ -433,7 +433,7 @@ pub struct NodeStyle {
 impl Default for NodeStyle {
     fn default() -> NodeStyle {
         NodeStyle {
-            text_color: Background::Solid(ColorOrVar::Color(Color::from_u8s(0, 0, 0, 255))),
+            text_color: Background::None,
```

Comment:
Why'd this change?

Reply:
The text only with strokes doesn't have a fill. But this defaults to solid black and will draw the fill as black color. So I changed it to default to `None`.